src/components/OptionRow.js

```diff
@@ -163,7 +163,7 @@ const OptionRow = (props) => {
                             {
                                 !_.isEmpty(props.option.icons)
                                 && (
-                                    props.option.shouldShowSubscript ? (
+                                    props.option.shouldShowSubscript && props.option.icons.length > 1 ? (
```

Comment:
This condition was added to prevent a warning message in the console, where the SubscriptAvatar is expecting 2 icons, but when the workspace is archived, there's only one (trash) icon.

Reply:
Hm, if the subscript avatar is always expecting two icons, then shouldn't the conditional be `.length === 2`?

Reply:
and isn't it weird how that logic kind of defeats the purpose of `shouldShowSubscript`?

Reply:
It seems to me that when the workspace is archived, then `shouldShowSubscript` should be `false` then?